.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+max-line-length = 127
+max-complexity = 13
+exclude = .git,__pycache__,doc/,docs/,build/,dist/,archive/,setup.py
+ignore = E121,E123,E126,E226,E24,E704,W503,W504
```

Comment:
for people not using `flake8`, could you add here comments specified what kind of errors/warnings are ignored. In other words, some text explaining what every single error/warning code means (or at least a link to the corresponding flake8 URL doc) would be extremly useful

Reply:
Did this now